### PR TITLE
ENHANCEMENT: Added watch movie command functionality 

### PR DESCRIPTION
**Pull request by JustenC14**
Added watch movie command functionality and fixed a bug in add movie where power users could not add directly to the movie super list.

### CODE REVIEW

File: src/commands/movie.js [hunk 1]
```diff
@@ -34,10 +34,17 @@ function commandsValid(args) {
  * @param {Object} message
  */
 function addMovie(args, message) {
+  let future;
+  if (args.future) {
+    future = 1;
+  } else {
+    future = 0;
+  }
+
```
Comment (ESattler) [inline src/commands/movie.js]:
```suggestion
const future = (args.future) ? 1 : 0;
```

File: src/helpers/movieTogetherAPI.js [hunk 2]
```diff
@@ -53,10 +60,11 @@ methods.callAddMovie = (body => callAPI('POST', '/movie', body));
 methods.callDeleteMovie = (body => callAPI('DELETE', '/movie', body));
 
 /**
- * Calls the GET /user/connected endpoint
+ * Calls the PATCH /movie endpoint
  * @param {Object} body
- * @returns {Promise}
+ * @returns {Priomise}
```
Comment (ESattler) [inline src/helpers/movieTogetherAPI.js]: Promise is misspelled

File: src/commands/movie.js [hunk 3]
```diff
@@ -34,10 +34,17 @@ function commandsValid(args) {
  * @param {Object} message
  */
 function addMovie(args, message) {
+  let future;
+  if (args.future) {
+    future = 1;
+  } else {
+    future = 0;
+  }
+const future = (args.future) ? 1 : 0;
```
Comment (ESattler) [inline src/commands/movie.js]: ya dingus, delete 37-43

Comment (JustenC14) [reply src/commands/movie.js]: Fixed

**Pull request merged**